src/HealthChecks.Nats/NatsHealthCheck.cs

```diff
@@ -0,0 +1,101 @@
+using System.Text;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using NATS.Client;
+
+namespace HealthChecks.Nats
+{
+    /// <summary>
+    /// Health check for Nats Server
+    /// </summary>
+    public sealed class NatsHealthCheck : IHealthCheck, IDisposable
+    {
+        /// <summary>
+        ///  Provides factory methods to create connections to NATS Servers.
```

Comment:
```suggestion
        /// Provides factory methods to create connections to NATS Servers.
```

Also note that you don't have to document private APIs. Better to document public APIs.